src/Analysis/GWASResults/Views/Results/Results.stories.jsx

```diff
@@ -73,6 +77,49 @@ MockedSuccess.parameters = {
   },
 };
 
+const dummyS3BucketLocation = "https://some-bucket.s3.amazonaws.com/gwas-workflow-123/test_pheweb.json";
+export const MockedSuccess2 = MockTemplate.bind({});
+MockedSuccess2.args = selectedRowData5;
+MockedSuccess2.parameters = {
+  msw: {
+    handlers: [
+      rest.get(
+        'http://:argowrapperpath/ga4gh/wes/v2/status/:workflowname',
+        (req, res, ctx) => {
+          const { argowrapperpath, workflowname } = req.params;
+          console.log(argowrapperpath);
+          console.log(workflowname);
+          return res(
+            ctx.delay(500),
+            ctx.json(WorkflowStatusResponse2)
+          );
+        }
+      ),
+      rest.get(
+        'http://:server/user/data/download/:index_did',
+        (req, res, ctx) => {
+          const { index_did } = req.params;
+          console.log(index_did);
+          return res(
+            ctx.delay(500),
+            ctx.json({"url": index_did === '222-8888-7777-bbbb123456-777777' ? dummyS3BucketLocation + "?X-Amz-Algorithm=AWS4-ETC" : "manhattanPheWebJsonFile.zip"}) // note: the .json and .zip here are fake urls
+          );
+        }
+      ),
+      rest.get(
+        dummyS3BucketLocation,
+        (req, res, ctx) => {
+          const { index_did } = req.params;
+          console.log(index_did);
+          return res(
+            ctx.delay(500),
+            ctx.json(manhattanPheWebJsonFile)
+          );
+        }
+      ),
+    ],
+  },
+};
 
 export const MockedError = MockTemplate.bind({});
```

Comment:
If we think it is possible the end user will encounter this error while using the application, it would be nicer if there was an error UI informing the user that something went wrong instead of crashing the application and/or showing a blank screen. I'm seeing this in the storybook: 
<img width="1382" alt="image" src="https://github.com/uc-cdis/data-portal/assets/113449836/9afcbfbf-65d1-4011-b095-0dbe4b434eb7">

Reply:
fixed, thanks.